classes/db/DAO.inc.php

```diff
@@ -406,7 +412,7 @@ function convertToDB($value, &$type) {
 		switch ($type) {
 			case 'object':
 			case 'array':
-				$value = serialize($value);
+				$value = json_encode($value);
```

Comment:
Does `JSON_UNESCAPED_UNICODE` need to be specified here too?

Reply:
Yes, you are right. Do you think other `JSON_UNESCAPED_...` options could be useful here? It all shouldn't affect decoding but I'm not experienced here.

Reply:
No, I'm not aware of others -- just spotted this one due to inconsistency with other `json_decode` calls in the PR.

Reply:
Yes, I've used it in the upgrade script according to the recommendations I found on the web; just forgot to put it here. Looking at the other options, I don't think that turning off escaping of line terminators or slashes will make any difference.